py/bitbox02/bitbox02/bootloader.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, transport: TransportLayer, device_info: DeviceInfo):
         self.expected_magic = {
             "bb02-bootloader": SIGDATA_MAGIC_STANDARD,
             "bb02btc-bootloader": SIGDATA_MAGIC_BTCONLY,
+            "bb02-base": SIGDATA_MAGIC_STANDARD,
```

Comment:
please add a `SIGDATA_MAGIC_BITBOXBASE_STANDARD = 0xab6bd345` (this is the new magic) and use it here.